src/screens/ParadeMapScreen/__snapshots__/Map.test.js.snap

```diff
@@ -1289,10 +1298,17 @@ exports[`Map component renders without location button when permission is restri
           },
         ]
       }
+      mapRef={
```

Comment:
I'm confused as to why mapRef is being passed to the TerminalMarkers props. Are we spreading props?

Reply:
ditto ^^